airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1953,6 +2025,10 @@ components:
           properties:
             withRefreshedCatalog:
               type: boolean
+            withOperations:
```

Comment:
i don't think you need the "with" here. It was added on the other field because it's a boolean so it is trying to use "with" to make that clear.